openedx_events/event_bus/avro/tests/test_deserializer.py

```diff
@@ -1,6 +1,7 @@
 """Tests for avro.deserializer"""
```

Comment:
can we add docstrings to each test you're adding? that'll help with the review.

Reply:
Done.